test/integration_test_runner.rb

```diff
@@ -0,0 +1,15 @@
+require 'test/unit'
+require_relative '../lib/pmdtester/runner'
+class IntegrationTestRunner < Test::Unit::TestCase
+  def test_runner
+    Process.fork do
+      argv = %w[-r target/repositories/pmd -b master -bc config/design.xml
+                -p pmd_releases/6.1.0 -pc config/design.xml -l test/resources/project-test.xml]
+
+      PmdTester::Runner.new(argv).run
+    end
+    Process.wait
+
+    assert_equal(0, $CHILD_STATUS.exitstatus)
```

Comment:
That's already very good.
We should additional check at least for the existence of the generated reports - so, we expect a diff report html file to be written.